doc/rados/operations/placement-groups.rst

```diff
@@ -664,31 +667,33 @@ Valid formats are ``plain`` (default) and ``json``.
 Get Statistics for Stuck PGs
 ============================
 
-To get the statistics for all placement groups stuck in a specified state,
-execute the following:
+To see the statistics for all PGs that are stuck in a specified state, run a
+command of the following form:
 
 .. prompt:: bash #
 
    ceph pg dump_stuck inactive|unclean|stale|undersized|degraded [--format <format>] [-t|--threshold <seconds>]
 
-**Inactive** Placement groups cannot process reads or writes because they are waiting for an OSD
-with the most up-to-date data to come up and in.
+- **Inactive** PGs cannot process reads or writes because they are waiting for
+  an OSD with the most up-to-date data to come ``up`` and ``in``.
 
-**Unclean** Placement groups contain objects that are not replicated the desired number
-of times. They should be recovering.
+- **Unclean** PGs contain objects that have not been replicated the desired
+  number of times. Under normal conditions, it can be assumed that these PGs
+  are recovering.
 
-**Stale** Placement groups are in an unknown state - the OSDs that host them have not
-reported to the monitor cluster in a while (configured by ``mon_osd_report_timeout``).
+- **Stale** PGs are in an unknown state -- the OSDs that host them have not
+  reported to the monitor cluster for a certain period of time (determined by
+  ``mon_osd_report_timeout``).
 
-Valid formats are ``plain`` (default) and ``json``. The threshold defines the minimum number
-of seconds the placement group is stuck before including it in the returned statistics
-(default 300 seconds).
+Valid formats are ``plain`` (default) and ``json``. The threshold defines the
+minimum number of seconds the PG is stuck before it is included in the returned
+statistics (default: 300 seconds).
```

Comment:
Suggest `(default: 300)` since the sentence already specifies the units.

Reply:
Yeah, you're right.

Accepted.